solr/core/src/java/org/apache/solr/schema/TextField.java

```diff
@@ -173,8 +173,10 @@ public static BytesRef analyzeMultiTerm(String field, String part, Analyzer anal
 
       TermToBytesRefAttribute termAtt = source.getAttribute(TermToBytesRefAttribute.class);
 
-      if (!source.incrementToken())
-        throw  new SolrException(SolrException.ErrorCode.BAD_REQUEST,"analyzer returned no terms for multiTerm term: " + part);
+      if (!source.incrementToken()) {
+        // Accept no token because it may have been filtered out by a StopFilter for example.
+        return null;
+      }
```

Comment:
Optional: Consider adding javadocs for TextField.analyzeMultiTerm().  Yes, someone else should have long ago